### PR TITLE
bug/skip_crash

### PR DESCRIPTION
**Pull request by dephillipsmichael**
Fixed bug with skipped question results (specifically choice answers)

### CODE REVIEW

File: researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/factory/ArchiveFileFactory.java [hunk 1]
```diff
@@ -69,7 +69,10 @@ public ArchiveFile fromResult(@NonNull Result result) {
         checkNotNull(result.getEndDate());
 
         if (result instanceof StepResult) {
-            return fromStepResult((StepResult) result);
+            // Skipped steps will have a null result
```
Comment (liujoshua) [inline researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/factory/ArchiveFileFactory.java]: Can we do this null handling in fromStepResult? 
We'd still need to check return value of fromStepResult to know whether to return early, but new code that calls fromStepResult without null checking first would no longer crash

**Comment by dephillipsmichael** [comment]
@liujoshua random issue, but see the CI, it looks like the 3rd party graph dependency may have been deleted.  Ideas on resolution?

File: researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/factory/ArchiveFileFactory.java (hunk 1 above)
Comment (dephillipsmichael) [reply researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/factory/ArchiveFileFactory.java]: Sure, I'll move it there.

**Review by liujoshua — APPROVED** [review]
Looks good. One suggestion

**Pull request merged**